resource_packs/packs.json

```diff
@@ -1081,6 +1081,11 @@
 					"name": "Translucent Pumpkin Overlay",
 					"description": "Increases visibility when wearing a Pumpkin"
 				},
+				{
+					"id": "no_fog",
+					"name": "No Fog",
+					"description": "Removes all fogs"
```

Comment:
Use the description from vanilla tweaks for more clarification (dont include the warning text for sodium, optifine, etc stuff cuz there is no need for it in bedrock)